src/frontend/screens/Library/components/CategoriesManager/index.tsx

```diff
@@ -30,6 +30,8 @@ function CategoryItem({
   const [renameMode, setRenameMode] = useState(false)
   const [newName, setNewName] = useState(name)
   const [removeMode, setRemoveMode] = useState(false)
+  const isNewNameEmptyOrEqualsOldName =
+    newName?.trim() === '' || newName === name
```

Comment:
do we need the `?` here and in `newCategoryName?.trim()`? they should be strings always, and if there's any case when they are possible undefined we should update the type to reflect that

Reply:
No, we don't need it at all.
I just added because I don't know the way if there is a bug or something somewhere that makes the input `undefined`, so I though that would be better have and don't need than need and don't have.
But I'll remove.

Reply:
Done.